scripts/clean_protos.sh

```diff
@@ -39,3 +39,6 @@ echo "Outputing protobuf files:"
   done < tmp
   rm tmp
 )
+
+echo $'\noption go_package = "github.com/iov-one/weave";' >> spec/gogo/codec.proto
+echo $'\noption go_package = "github.com/iov-one/weave";' >> spec/proto/codec.proto
```

Comment:
I think those lines are not needed.

Reply:
These lines add to other committed files as spec is being auto-generated along with the docs.